wiki/FAQ/en.md

```diff
@@ -56,154 +62,148 @@ _[Osu! Tatakae! Ouendan][ouendan wikipedia]_ is a rhythm game for the Nintendo D
 
 Instead of clicking circles, you are now tasked to move the catcher at the bottom of the screen while the fruits fall from above.
 
 #### Controlling the catcher
-If the [Relax][Relax wikilink] mod is used, control of the catcher is done by moving the mouse instead.
--   By default, the left \[&lt;-\] and right \[-&gt;\] arrow keys are used to move the catcher.
--   Pressing the left \[SHIFT\] key with one of the arrow keys will make the catcher dash in that direction at x1.5 speed (indicated by a bright glow at the catcher).
+-   By default, the left `&lt;-` and right `-&gt;` arrow keys are used to move the catcher.
+-   Pressing the left `Shift` key with one of the arrow keys will make the catcher _dash_ in that direction at x1.5 speed (as indicated with a **bright glow** from the catcher).
+
+If you enable the [Relax][Relax wikilink] mod, you can control the catcher with your in-game cursor instead. However, this will make your play unrankable.
 
-### What is "osu!mania"?
+### What is osu!mania?
 _Full explanation: [osu!mania][osu!mania wikilink]_
 
 ![osu!mania Interface][osu!mania Interface image]
 
-_osu!mania_ is the third special game mode to be added in, after osu!taiko and osu!catch game mode.
+osu!mania is the third game mode added to osu!, after osu!taiko and osu!catch game modes.
 
-This mode has been widely used in almost all of the major rhythm games that require good hand/leg coordination, where the beat notes with their quantity depending on BPM and difficulty move on the conveyer, and the player have to press the correct key for that specific beat note in time. Though the game mode was originally made as an imitation of Beatmania style of playing, however, by changing the number of keys, you can make it look more like Guitar Hero (5) or Dance Dance Revolution (4) and so on.
+This mode has been widely used in almost all of the major rhythm games. It require good hand and/or leg coordination where the notes (with their quantity depending on BPM and difficulty) move on a conveyer. With this, the player will have to press the correct key for that specific note in time. Though the game mode was originally made as an imitation of the Beatmania style of playing, osu!mania allows changing number of keys you play with (this means you can make it look more like Guitar Hero \[5 keys\] or Dance Dance Revolution \[4 keys\] and so on).
 
-Currently, you can set from 1 to 9 Key by game modifier (by default, it would be around 4 to 8 Key, except for osu!mania-specific beatmaps) to an upward of 2 - 18 Key (two playfield) with the [Co-Op][Co-Op wikilink] game modifier.
+Currently, you can manually set from 1 to 9 Key by the [xK][xK wikilink] game modifier (when converting from osu!standard, the default key range would be around 4 to 7 Key, except for osu!mania-specific beatmaps) to an upward of 2 to 18 Key (two playfield) with the [Co-Op][Co-Op wikilink] game modifier.
```

Comment:
This reads more oddly than ever before. Anyways, why capitalize key and make it singular? I assume that `Key = xK mod` but you've already used xK mod in place of Key, so this defeats the purpose of trying to treat Key as a name (or a proper noun). I suggest something like this:
>When converting a beatmap from osu!standard, the default key range would be around 4 to 7 keys. With the [xK][xK wikilink] game modifier, you can manually set the number of keys you play with 1 to a total of 9 keys with a score multiplier reduction. However, this will not work with osu!mania-specific beatmaps. With the [Co-Op][Co-Op wikilink] game modifier, you can split the stage in two and play with 2 to a total of 18 keys with a score multiplier reduction. Similarly to the xK game modifier, using the Co-Op game modifier with an osu!mania-specific beatmap will not double the number of keys; instead, it will divide the stage in two.